packages/sdk/src/types/requests/ConnectorResponse.ts

```diff
@@ -0,0 +1,25 @@
+export interface ConnectorResponse {
+    "@type": "Response";
+    result: "Accepted" | "Rejected";
+    requestId: string;
+    items: (ConnectorResponseItemGroup | ConnectorResponseItem)[];
+}
+
+export interface ConnectorResponseItem {
+    "@type": string;
+    result: ConnectorResponseItemResult;
+    metadata?: object;
```

Comment:
Do we have metadata in responses? A quick runtime search says no

Reply:
nope, it's a relic